DorisProject/src/app/pages/add-category/add-category.component.ts

```diff
@@ -71,4 +88,20 @@ export class AddCategoryComponent implements OnInit {
     return this.addCategoryForm.get('description').invalid && this.addCategoryForm.get('description').touched;
   }
 
+
+  openModal(request: string) {
+    const dialogRef = this.dialog.open(ModalComponent, {
+      data: {textrequest: request, textresponse: this.responseModal,nameService:'Category'}
+    });
+    dialogRef.afterClosed().subscribe(result => {
```

Comment:
Codacy has a fix for the issue: [Replace `result` with `(result)`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
    dialogRef.afterClosed().subscribe((result) => {
```